.github/workflows/smoke-test-ipv6.yaml

```diff
@@ -37,21 +47,18 @@ jobs:
           sudo sysctl net.ipv6.conf.all.forwarding=1
           sudo systemctl restart docker
 
-      - name: Build docker images
-        run: |
-          make docker-cilium-image
-          make docker-operator-generic-image
-
       - name: Create kind cluster
         uses: helm/kind-action@v1.1.0
         with:
           version: ${{ env.KIND_VERSION }}
           config: ${{ env.KIND_CONFIG }}
 
-      - name: Load local images into kind cluster
+      - name: Wait for images to be available
+        timeout-minutes: 10
+        shell: bash
         run: |
-          kind load docker-image --name chart-testing quay.io/cilium/cilium:latest
-          kind load docker-image --name chart-testing quay.io/cilium/operator-generic:latest
+          until curl --silent -f -lSL "https://quay.io/api/v1/repository/${{ github.repository_owner }}/cilium-ci/tag/${{ steps.vars.outputs.tag }}/images" &> /dev/null; do sleep 45s; done
+          until curl --silent -f -lSL "https://quay.io/api/v1/repository/${{ github.repository_owner }}/operator-ci/tag/${{ steps.vars.outputs.tag }}/images" &> /dev/null; do sleep 45s; done
```

Comment:
Can this hang forever?

Reply:
I had to check my changes again because I added the `timeout-minutes` in line 57 and I got confused by this question :sweat_smile: . Isn't the timeout enough?